desktop/src/main/java/bisq/desktop/main/account/content/altcoinaccounts/AltCoinAccountsView.java

```diff
@@ -173,7 +173,6 @@ protected void buildForm() {
 
         Tuple3<Label, ListView<PaymentAccount>, VBox> tuple = addTopLabelListView(root, gridRow, Res.get("account.altcoin.yourAltcoinAccounts"), Layout.FIRST_ROW_DISTANCE);
         paymentAccountsListView = tuple.second;
-        paymentAccountsListView.setPrefHeight(2 * Layout.LIST_ROW_HEIGHT + 14);
         setPaymentAccountsCellFactory();
```

Comment:
By removing that it would require a min. height as otherwise the list can get truncated.
<img width="1202" alt="Screen Shot 2019-08-14 at 18 24 35" src="https://user-images.githubusercontent.com/1449498/63038237-fe3e3a00-bec0-11e9-8d6c-72ec727aef4c.png">

Reply:
paymentAccountsListView.setMinHeight(2 * Layout.LIST_ROW_HEIGHT + 14);
will do it.
Same on fiat account view

Reply:
done